frontend/src/components/ui/calendar.tsx

```diff
@@ -0,0 +1,66 @@
+"use client"
+
+import * as React from "react"
+import { ChevronLeft, ChevronRight } from "lucide-react"
+import { DayPicker } from "react-day-picker"
+
+import { cn } from "@/lib/utils"
+import { buttonVariants } from "@/components/ui/button"
+
+export type CalendarProps = React.ComponentProps<typeof DayPicker>
+
+function Calendar({
+  className,
+  classNames,
+  showOutsideDays = true,
+  ...props
```

Comment:
Spreading props requires investigation and brainpower to understand what they are, I prefer having explicit props everywhere, even if they are repeated

Reply:
this is a shadcn/ui component.